_posts/2016-02-10-installing-server-environment-with-powershell.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+title: Installing server environment with PowerShell
+author: Rinorragi
+excerpt: Automize your IIS webserver installation with PowerShell 
+categories: 
+- EPiServer
+tags: 
+- EPiServer 
+- DOTNET 
+- PowerShell
+---
+Most likely as a reader you are mostly interested about my scripts. Here are our scripts for setting up Windows Server 2012 R2 ready for WebDeployments and running EPiServer ASP.NET site!
```

Comment:
interested about -> interested in.
my scripts - seuraavassa lauseessa our scripts ?